tests/test_routes.py

```diff
@@ -254,7 +254,7 @@ def handler2(request):
 
 
 def test_route_invalid_parameter_syntax(app):
-    with pytest.raises(ValueError):
+    with pytest.raises(InvalidUsage):
```

Comment:
Why does this raise an error for the client and not an error for the developer? I'd expect it to raise when declaring the route, not when accessing it. Am I missing something? 🤔

Reply:
It raises at run time. Which we are just using the client as a quick and dirty proxy for running. Perhaps not the cleanest test.